qiwis.py

```diff
@@ -248,24 +248,25 @@ def load(self, appInfos: Mapping[str, AppInfo]):
             self.createApp(name, info)
         logger.info("Loaded %d app(s)", len(appInfos))
 
-    def addFrame(self, name: str, frame: QWidget, info: AppInfo):
+    def addFrame(self, name: str, title: str, frame: QWidget, info: AppInfo):
         """Adds the given frame and wraps it with a wrapper widget.
 
         This is not a qiwiscall because QWidget is not Serializable.
         
         Args:
-            name: A name of app.
-            frame: A frame to show.
-            info: An AppInfo object describing the app.
+            name: The app name to which frame is added.
+            titme: The frame title.
+            frame: Frame object to show.
+            info: AppInfo object describing the app.
         """
         if info.pos == "center":
             wrapperWidget = MdiSubWindow(self.centralWidget)
-            wrapperWidget.setWindowTitle(name)
+            wrapperWidget.setWindowTitle(title)

```

Comment:
How about keeping the app name?
For example, `f"{name} - {title}"`, `f"{title} ({name})"`, etc.
I think we should be able to know the app name of each frame.

Reply:
In addition, it would be good to have an empty title - whose actual title is just the app name.

Reply:
I see! Then, I will adopt the first suggestion, but if an empty title, I will show only the app name.